From 6a47d2548fade42aa36022d03453a3077d65fbba Mon Sep 17 00:00:00 2001
From: Julien Olivain <julien.olivain@nxp.com>
Date: Tue, 12 Nov 2019 18:07:14 +0100
Subject: [PATCH 12/37] MLK-22952 mxc: vpu_windsor: add missing semicolon

commit 4c2207e837d7d7bbc45a2d5644462cfdd87febee from
https://source.codeaurora.org/external/imx/linux-imx.git

Add a missing semicolon. This missing semicolon is not breaking the
build in defconfig since the next MODULE_DEVICE_TABLE() expand to an
empty string, so the next semicolon is used. When this macro expands
to a non-empty string, the build breaks. This was found by a
"make allmodconfig" build.

Signed-off-by: Julien Olivain <julien.olivain@nxp.com>
Reviewed-by: Shijie Qin <shijie.qin@nxp.com>
Acked-by: Zhou Peng <eagle.zhou@nxp.com>
(cherry picked from commit d36d9e1942fc335262672ed4e7c5f102ac0ca8f2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/vpu_windsor/vpu_encoder_b0.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/vpu_windsor/vpu_encoder_b0.c b/drivers/mxc/vpu_windsor/vpu_encoder_b0.c
index df269b3138cb..5e0d7f47c9a1 100644
--- a/drivers/mxc/vpu_windsor/vpu_encoder_b0.c
+++ b/drivers/mxc/vpu_windsor/vpu_encoder_b0.c
@@ -5889,7 +5889,7 @@ static const struct of_device_id vpu_enc_of_match[] = {
 	  .data = (void *)&supported_plat_types[IMX8QXP]
 	},
 	{}
-}
+};
 MODULE_DEVICE_TABLE(of, vpu_enc_of_match);
 
 static struct platform_driver vpu_enc_driver = {
-- 
2.17.1

